README.md

```diff
@@ -17,7 +17,9 @@ features for JSON Pointers as well.
 
 ## Documentation
 
-The documentation for this project can be found here: https://github.com/whitlockjc/json-refs/blob/master/docs/README.md
+The documentation for this project can be found here: https://github.com/whitlockjc/json-refs/blob/master/docs/README.md .
+
+The API can be found at https://github.com/whitlockjc/json-refs/blob/master/docs/API.md .
```

Comment:
If you want I can change the second one to end with "here" too, but I think that "at" should not be followed by a colon because as a preposition it should join with the following noun (the URL), whereas the ending on "here" is pausing after a complete phrase to introduce the URL.

Regardless of what you want for that, I think we should surround the URLs here with `<>` to avoid the extra spaces (and also maybe for the sake of the markdown linter I have in Atom which always suggests doing this).